manim/mobject/opengl/opengl_mobject.py

```diff
@@ -1673,7 +1738,12 @@ def hierarchical_model_matrix(self):
             current_object = current_object.parent
         return np.linalg.multi_dot(list(reversed(model_matrices)))
 
-    def wag(self, direction=RIGHT, axis=DOWN, wag_factor=1.0):
+    def wag(
```

Comment:
Just noticed, this method was deleted in Cairo. What do you think about deleting it (either in this PR or a different one)?

Reply:
I noticed it as well, and I think this should be deleted in another PR